mpt/impl/imap-mailbox/core/src/main/resources/org/apache/james/imap/scripts/Create.test

```diff
@@ -58,6 +58,18 @@ S: 18 OK LIST completed.
 C: 19 CREATE alpha
```

Comment:
Is there any way we can have in the message returned to the user that the mailbox name was too long ?